modules/aws/ssm.go

```diff
@@ -68,3 +73,127 @@ func NewSsmClientE(t testing.TestingT, region string) (*ssm.SSM, error) {
 
 	return ssm.New(sess), nil
 }
+
+// WaitForSsmInstanceE waits until the instance get registered to the SSM inventory.
+func WaitForSsmInstanceE(t testing.TestingT, awsRegion, instanceID string, timeout time.Duration) error {
+	timeBetweenRetries := 2 * time.Second
+	maxRetries := int(timeout.Seconds() / timeBetweenRetries.Seconds())
```

Comment:
If someone sets `timeout` to 1 second, won't this end up with a `maxRetries` of 0?

Reply:
Yes, so `DoWithRetryE()` will try once and return an error if it did not succeed which is what is expected right?

Reply:
Ah, right. For some reason, I had in my mind that `maxRetries` of 0 means `DoWithRetry` won't run at all, but looking at it again, I realize it will run, but won't retry, which is what we want here. So all is well.